frontend/common/src/helpers/localize.test.ts

```diff
@@ -44,5 +49,16 @@ describe("localize helper tests", () => {
       expect(localizePath("admin/users", "en")).toBe("en/admin/users");
       expect(localizePath("fr/admin/users", "en")).toBe("en/admin/users");
     });
+    test("works on single slash", () => {
+      expect(localizePath("/", "en")).toBe("/en/");
```

Comment:
So then, is this going to fail if it _doesn't_ expect a trailing slash?
```suggestion
      expect(localizePath("/", "en")).toBe("/en");
```

Reply:
So, this specific test is a bit weird and different from the others.  I think these screenshots (from the handy debugger 😁 ) help explain why.  In a nutshell, the `split` function assigns an empty string _before and after_ the slash.
A regular case:
![image](https://user-images.githubusercontent.com/8978655/151442597-e53fd0fe-fd2a-4380-b4c5-d7fc48b08845.png)
The single slash case:
![image](https://user-images.githubusercontent.com/8978655/151442622-cfd455a3-6af9-4c9f-8ff3-e867318cddf4.png)